Frontend/src/data/rental/dataHooks.js

```diff
@@ -0,0 +1,21 @@
+import Database from "../../database/ENV_DATABASE";
+
+export default {
+    async onDataLoaded(data) {
+        const itemIds = data.map(d => d.item_id)
+        const customerIds = data.map(d => d.customer_id)
+        
+        const results = await Promise.all([
+            Database.fetchDocsBySelector({ $and: [ { type: 'item', id: { $or: itemIds } } ] }, ['id', '_rev', 'highlight'], [], data.length * 10),
+            Database.fetchDocsBySelector({ $and: [ { type: 'customer', id: { $or: customerIds } } ] }, ['id', '_rev', 'highlight'], [], data.length * 10),
+        ])
+        data.forEach(r => {
+            results[0].sort((a, b) => b._rev.localeCompare(a._rev))
+            results[1].sort((a, b) => b._rev.localeCompare(a._rev))
```

Comment:
I don't understand these two lines. Why do you sort by _rev? Also I think you could move these lines out of the forEach.

Reply:
Sorting by `_rev` in descending order ensures the most recent version of the document is used in the lines below.

Reply:
But isn't only the most recent version of any document returned anyway?

Reply:
No, when I tested multiple documents were returned. The previous code did `.find()` (see [here](https://github.com/leih-lokal/LeihLokalVerwaltung/blob/8caf45e53ca78f763868dc987f8f05eb3a57951f/Frontend/src/data/rental/columns.js#L42), for example), effectively taking the first one, whichever that is. I'm pretty sure this was a bug in the current version.

Reply:
Hm, I still can't really believe that it also returns old revisions of documents. I also could not find this behavior documented anywhere. 

But I wrote a short script to test that PouchDB.find does not return old revisions of documents and for me it does not:
```js
const PouchDB = require('pouchdb');
PouchDB.plugin(require('pouchdb-find'));

const db = new PouchDB('mydb');

db.put({ _id: 'doc1', name: 'Mario', age: 30 })
  .then(() => db.get('doc1'))
  .then((existingDoc) => {
    existingDoc.age = 31;
    return db.put(existingDoc);
  })
  .then(() => {
    return db.find({
		include_revs: true,
      selector: { name: 'Mario' },
      fields: ['_id', 'name', 'age', '_rev'],
    });
  })
  .then(console.log)
```

Reply:
Yeah, I think you're right, sorry. I must have confused something on the way. Removed the sorting part.